### PR TITLE
Add new cli documentation

### PR DESCRIPTION
**Pull request by eric-pigeon**

### CODE REVIEW

File: docs/source/cli/workareactl.html.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+title: workareactl
+body_class: body--nude
+---
+## workareactl
+
+A tool for interacting with workarea-commerce environments
```
Comment (bencrouse) [inline docs/source/cli/workareactl.html.md:7]:
```suggestion
A tool for interacting with Workarea Commerce Cloud environments
```

File: docs/source/cli/workareactl_edit.html.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+title: workareactl edit
+body_class: body--nude
+---
+## workareactl edit
+
+edit various aspects of the environment
```
Comment (bencrouse) [inline docs/source/cli/workareactl_edit.html.md:7]:
```suggestion
Edit various aspects of the environment
```

File: docs/source/cli/workareactl_edit.html.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+### Synopsis
+
+edit various aspects of the environment
```
Comment (bencrouse) [inline docs/source/cli/workareactl_edit.html.md:11]:
```suggestion
Edit various aspects of the environment
```

File: docs/source/cli/workareactl_edit_proxy.html.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+---
+title: workareactl edit proxy
+body_class: body--nude
+---
+## workareactl edit proxy
+
+edit the proxy
```
Comment (bencrouse) [inline docs/source/cli/workareactl_edit_proxy.html.md:7]:
```suggestion
Edit the proxy
```

File: docs/source/cli/workareactl_edit_proxy.html.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+### Synopsis
+
+edit the proxy
```
Comment (bencrouse) [inline docs/source/cli/workareactl_edit_proxy.html.md:11]:
```suggestion
Edit the proxy
```

File: docs/source/cli/workareactl_edit_secrets.html.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
+---
+title: workareactl edit secrets
+body_class: body--nude
+---
+## workareactl edit secrets
+
+edit the secrets
```
Comment (bencrouse) [inline docs/source/cli/workareactl_edit_secrets.html.md:7]:
```suggestion
Edit the secrets
```

File: docs/source/cli/workareactl_edit_secrets.html.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+### Synopsis
+
+edit the secrets
```
Comment (bencrouse) [inline docs/source/cli/workareactl_edit_secrets.html.md:11]:
```suggestion
Edit the secrets
```

File: docs/source/index.html.erb [hunk 8]
```diff
@@ -87,12 +87,12 @@
...
     <div class='align-center'>
-      <% link_to '/cli.html', class: 'box' do %>
+      <% link_to '/commerce-cloud.html', class: 'box' do %>
         <div class='box__header'>
-          <span class='box__heading'>CLI</span>
+          <span class='box__heading'>Commerce Cloud</span>
         </div>
         <p class='box__text'>
-          Make life easy with our command line interface!
+          Information about our hosted commerce cloud solution.
```
Comment (bencrouse) [inline docs/source/index.html.erb:95]:
```suggestion
          Information about our hosted Commerce Cloud solution.
```

**Review by bencrouse — CHANGES_REQUESTED** [review]
A couple comments:
* Do we need `docs/source/cli-old.html.md` for anything? Let's remove if it's cruft, we have it in the git history if we need to revisit.
* Conventionally, the docs article files use dashes, not underscores, can you update for consistency?
* It would be easy and consistent for branding to do find and replace `workarea-commerce environments` => `Workarea Commerce Cloud environments`

**Comment by eric-pigeon** [comment]
I am still linking to the old cli docs, I think we can remove it after everyone is migrated to the new cli.

**Review by bencrouse — APPROVED** [review]
@mcfearsome @MMartyn @wmferri do you guys want to take a look before I merge this?

**Review by MMartyn — APPROVED** [review]

**Pull request merged**